AEPIdentity/Sources/IdentityState.swift

```diff
@@ -269,14 +269,7 @@ class IdentityState {
         identityProperties.privacyStatus = newPrivacyStatus
 
         if newPrivacyStatus == .optedOut {
-            identityProperties.ecid = nil
-            identityProperties.advertisingIdentifier = nil
-            identityProperties.blob = nil
-            identityProperties.locationHint = nil
-            identityProperties.customerIds?.removeAll()
-            identityProperties.isAidSynced = false
-            identityProperties.pushIdentifier = nil
-            pushIdManager.updatePushId(pushId: nil)
```

Comment:
This call to `pushIdManager.updatePushId()` is replaced with `pushIdManager.resetPersistedFlags()` in the function `clearIdentifiers()`.